tests/app/views/contexts/test_list_collector.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+from app.views.contexts import list_collector
+
+
+def test_build_list_collector_context(
+    rendered_block, answer_store, list_store, form, app
+):
+    context = list_collector.build_list_collector_context(
+        rendered_block, answer_store, list_store, 'en', form
+    )
+
+    assert {'list_items', 'add_link'} <= {*context.keys()}
+    assert context['add_link'] == '/questionnaire/people/list-collector/'
+
+
+def test_build_list_collector_context_no_summary(
+    rendered_block, answer_store, list_store, form, app
+):
+    del rendered_block['summary']
+    context = list_collector.build_list_collector_context(
+        rendered_block, answer_store, list_store, 'en', form
+    )
+
+    assert context['list_items'] == []
+
+
+def test_build_list_items_summary_context(
+    rendered_block, answer_store, list_store, app
+):
+    list_item_context = list_collector.build_list_items_summary_context(
+        rendered_block, answer_store, list_store, 'en'
+    )
+    item_titles = [list_item['item_title'] for list_item in list_item_context]
+
+    assert len(list_item_context) == 3
+    for list_item in list_item_context:
+        assert {
+            'answers',
+            'item_title',
+            'edit_link',
+            'remove_link',
+            'primary_person',
+        } <= {*list_item.keys()}
+    assert 'Toni Morrison' == item_titles[0]
+    assert '' == item_titles[1]
+
+
+def test_assert_primary_person_string_appended(
+    rendered_block, answer_store, list_store, app
+):
+    list_store['people'].primary_person = 'PlwgoG'
+    list_item_context = list_collector.build_list_items_summary_context(
+        rendered_block, answer_store, list_store, 'en'
+    )
+
+    assert list_item_context[0]['primary_person'] == True
+    assert 'Toni Morrison (You)' == list_item_context[0]['item_title']
```

Comment:
Should this test also check that other list items do not have `(You)` appended to them?